src/easydiffraction/analysis/fitting/results.py

```diff
@@ -0,0 +1,126 @@
+    ) -> None:
+        status_icon = '✅' if self.success else '❌'
+        rf = rf2 = wr = br = None
+        if y_obs is not None and y_calc is not None:
+            rf = calculate_r_factor(y_obs, y_calc) * 100
+            rf2 = calculate_r_factor_squared(y_obs, y_calc) * 100
+        if y_obs is not None and y_calc is not None and y_err is not None:
+            wr = calculate_weighted_r_factor(y_obs, y_calc, y_err) * 100
+        if f_obs is not None and f_calc is not None:
+            br = calculate_rb_factor(f_obs, f_calc) * 100
+
+        print(paragraph('Fit results'))
+        print(f'{status_icon} Success: {self.success}')
+        print(f'⏱️ Fitting time: {self.fitting_time:.2f} seconds')
+        print(f'📏 Goodness-of-fit (reduced χ²): {self.reduced_chi_square:.2f}')
+        if rf is not None:
+            print(f'📏 R-factor (Rf): {rf:.2f}%')
+        if rf2 is not None:
+            print(f'📏 R-factor squared (Rf²): {rf2:.2f}%')
+        if wr is not None:
+            print(f'📏 Weighted R-factor (wR): {wr:.2f}%')
+        if br is not None:
+            print(f'📏 Bragg R-factor (BR): {br:.2f}%')
+        print('📈 Fitted parameters:')
+
+        headers = [
+            'datablock',
+            'category',
+            'entry',
+            'parameter',
+            'start',
+            'fitted',
+            'uncertainty',
+            'units',
+            'change',
+        ]
+        alignments = [
+            'left',
+            'left',
+            'left',
+            'left',
+            'right',
+            'right',
+            'right',
+            'left',
+            'right',
+        ]
+
+        rows = []
+        for param in self.parameters:
+            datablock_id = getattr(param, 'datablock_id', 'N/A')
+            category_key = getattr(param, 'category_key', 'N/A')
+            collection_entry_id = getattr(param, 'collection_entry_id', 'N/A')
+            name = getattr(param, 'name', 'N/A')
+            start = f'{getattr(param, "start_value", "N/A"):.4f}' if param.start_value is not None else 'N/A'
+            fitted = f'{param.value:.4f}' if param.value is not None else 'N/A'
+            uncertainty = f'{param.uncertainty:.4f}' if param.uncertainty is not None else 'N/A'
+            units = getattr(param, 'units', 'N/A')
+
+            if param.start_value and param.value:
+                change = ((param.value - param.start_value) / param.start_value) * 100
+                arrow = '↑' if change > 0 else '↓'
+                relative_change = f'{abs(change):.2f} % {arrow}'
+            else:
+                relative_change = 'N/A'
+
+            rows.append(
+                [datablock_id, category_key, collection_entry_id, name, start, fitted, uncertainty, units, relative_change]
+            )
+
+        render_table(
+            columns_headers=headers,
+            columns_alignment=alignments,
+            columns_data=rows,
+            show_index=True,
+        )
```

Comment:
I assume the code hasn't changed during the move, so not testing

Reply:
Right, there was no change in the code apart from the move.